ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/system_l1.schema.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  system_l1:
```

Comment:
We already have a system key, where l1 could just be another option.

Reply:
I separated it since `system l1` is in it's on section in the CLI.

Reply:
addressed in commit: 46f87f0